storage/src/checker/mod.rs

```diff
@@ -167,12 +180,21 @@ impl<S: WritableStorage> NodeStore<Committed, S> {
         let _leaked_areas = self.split_all_leaked_ranges(leaked_ranges, opt.progress_bar.as_ref());
         // TODO: add leaked areas to the free list
 
-        Ok(CheckerReport {
+        let physical_bytes = match self.physical_size() {
+            Ok(physical_bytes) => physical_bytes,
+            Err(e) => {
+                errors.push(e.into());
+                0
+            }
+        };
```

Comment:
I thought we weren't going to do any physical size checks? This is bad if we use a raw device.

Reply:
We are not doing any physical size checks. This is purely for collecting statistics.